pwa/src/templates/templateParts/filters/FiltersTemplate.tsx

```diff
@@ -58,6 +62,7 @@ export const FiltersTemplate: React.FC<FiltersTemplateProps> = ({ isLoading }) =
           placeholder="Publicatietype"
           defaultValue={TEMP_PUBLICATION_TYPES.find((option) => option.value === filters.publicationType)}
           isClearable
+          disabled
```

Comment:
Do we know when this will be enabled again? (Soft) removing the entire select field might be more user friendly for the time being.